### ISSUE TITLE
State not being restored.

### ISSUE THREAD

**Issue by lordzouga** [championswimmer/vuex-persist#13]
I have a list of objects that is stored in vuex. I have setup vuex-persist as instructed but on startup, it doesn't auto restore the state.

I have an action that runs every time the app is started, this action fetches data based on a variable that should be persisted with vuex-persist. 

```javascript
    loadJobs: function ({ commit }) {
      axios({
          method: 'get',
          url: 'http://localhost:9000/document/jobs',
          headers: {'X-BB-SESSION': 'secret-string'},
          params: {
            where: 'timeposted > ' + this.state.lastUpdate //this should have persisted value,
            page: 0,
            recordsPerPage: 50
          }
      })
      .then((response) => {
        if (response.data.data.length > 0){
           console.log(this.state.lastUpdate) // weirdly has the persisted value here
          commit('setJobs', { list: response.data.data })
        }
        commit('setLastUpdateTime', Math.floor(Date.now() / 1000));
      })
      .catch((error) => {
          console.log(error)
      })
  }
```

inside the response callback of the axios request, `this.state.lastUpdate` seems to have the right persisted value. other state attributes have this behavior as well.

my question being, when is the state being restored? is there a special scope needed before it can be accessed?

in some cases, I have to mutate a certain state before the persisted version is read by the bound vue component.

Any help will be appreciated. I feel as if I am missing something obvious here

**Comment by championswimmer** [comment]
could you show the source of your vuex store entry point

**Comment by msaelices** [comment]
Maybe this would helps. I had the same problem and this could be a workaroud. 

    mounted() {
      setTimeout(() => {
        this.$forceUpdate();
      }, 100);
    },

**Comment by msaelices** [comment]
If you want to know about my fix context. See https://github.com/msaelices/vue-onsenui-weather/commit/d34395b2c9a1bc098b30982267e8638009154970

**Comment by championswimmer** [comment]
this is the same problem as #15 

it has been fixed in v1.0
